gammapy/modeling/models/core.py

```diff
@@ -261,6 +242,61 @@ def reassign(self, datasets_names, new_datasets_names):
         return model
 
 
+class Model:
+    """Model class that contains only methods to create a model listed in the registries."""
+
+    @staticmethod
+    def create(tag, model_type=None, *args, **kwargs):
+        """Create a model instance.
+
+        Examples
+        --------
+        >>> from gammapy.modeling.models import Model
+        >>> spectral_model = Model.create("pl-2", model_type="spectral", amplitude="1e-10 cm-2 s-1", index=3)
+        >>> type(spectral_model)
+        <class 'gammapy.modeling.models.spectral.PowerLaw2SpectralModel'>
+        """
+        from . import (
+            MODEL_REGISTRY,
+            SPATIAL_MODEL_REGISTRY,
+            SPECTRAL_MODEL_REGISTRY,
+            TEMPORAL_MODEL_REGISTRY,
+        )
+
+        if model_type is None:
+            cls = MODEL_REGISTRY.get_cls(tag)
+        else:
+            registry = {
+                "spatial": SPATIAL_MODEL_REGISTRY,
+                "spectral": SPECTRAL_MODEL_REGISTRY,
+                "temporal": TEMPORAL_MODEL_REGISTRY,
+            }
+            cls = registry[model_type].get_cls(tag)
+        return cls(*args, **kwargs)
+
+    @staticmethod
+    def from_dict(data):
```

Comment:
I guess one could go through either `.create()` or `.from_dict()` in one of the cases to avoid the code duplication with the registries?

Reply:
added a function `_get_model_class_from_dict()` that is called by `.create()` and `.from_dict()`